addons/skin.estuary/xml/Home.xml

```diff
@@ -2,6 +2,8 @@
 <window>
 	<defaultcontrol>9000</defaultcontrol>
 	<backgroundcolor>background</backgroundcolor>
+	<onload condition="String.IsEmpty(Skin.String(pvr_tv_action))">Skin.SetString(pvr_tv_action,0)</onload>
+	<onload condition="String.IsEmpty(Skin.String(pvr_radio_action))">Skin.SetString(pvr_radio_action,0)</onload>
```

Comment:
this is not 100% fail-safe.
a user may have configured a different startup window (Settings > Interface > Startup),
if the user then navigates to Settings without going through the Home window,
the skin setting value will be undefined.

i think it's better to drop this and add `String.IsEmpty()`checks everywhere else.

Reply:
> this is not 100% fail-safe. a user may have configured a different startup window (Settings > Interface > Startup), if the user then navigates to Settings without going through the Home window, the skin setting value will be undefined.
> 
> i think it's better to drop this and add `String.IsEmpty()`checks everywhere else.

OK I now treat the empty value as 0.

Reply:
Another option if you want to use the empty state is I remove one state so it would be empty, 0, 1 or even text (English).  I was using the numbers to show states were incremented and decremented, now it is not as clear.